app/services/provider_interface/application_choice_presenter.rb

```diff
@@ -0,0 +1,33 @@
+module ProviderInterface
+  class ApplicationChoicePresenter
+    attr_reader :application_choice
+
+    def initialize(application_choice)
+      @application_choice = application_choice
+    end
+
+    def full_name
+      "#{application_choice.application_form.first_name} #{application_choice.application_form.last_name}"
+    end
+
+    def course_name
+      "Course #{application_choice.course_ucas_code}"
+    end
+
+    def status_name
+      # TODO: add proper human readable names for all states
+      application_choice.status.to_s.humanize
+    end
+
+    def updated_at
+      application_choice.updated_at
+    end
+
+    def to_param
+      # For now, the ID used in the URL for an "application" is the ID of
+      # the ApplicationChoice. This needs to be replaced by the 10 character
+      # string that we promised in the API spec.
+      application_choice.id.to_s
```

Comment:
This will be added in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/299 :)